google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/ReadModifyWriteRow.java

```diff
@@ -33,14 +33,18 @@
 public final class ReadModifyWriteRow implements Serializable {
   private static final long serialVersionUID = -8150045424541029193L;
 
-  private final String tableId;
+  private final TargetId targetId;
   private transient ReadModifyWriteRowRequest.Builder builder =
       ReadModifyWriteRowRequest.newBuilder();
 
   private ReadModifyWriteRow(@Nonnull String tableId, @Nonnull ByteString key) {
-    Preconditions.checkNotNull(tableId, "tableId can't be null.");
+    this(TableId.of(tableId), key);
```

Comment:
Same for this class, remove this ctor, no need for `Nonnull`, and add `@Deprecated` annotation with some comments.